<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogCadOpen</class>
 <widget class="QDialog" name="dialogCadOpen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dialogCadOpen</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Open CAD file</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="northEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="eastEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="elevEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="relButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Elevation relative to local map elevation</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="absButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Elevation is absolute</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Location on map of the CAD model origini</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Northing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Easting</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Elev (m)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rotxEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="rotyEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="rotzEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>in map projection coordinate (m)</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>50</y>
     <width>20</width>
     <height>181</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Model rotations</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>X (deg)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Y (deg)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Z (deg)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x is East, y is North, z is Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale factor</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scaleEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogCadOpen</receiver>
   <slot>doBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogCadOpen</receiver>
   <slot>doLoad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogCadOpen</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doBrowse()</slot>
  <slot>doLoad()</slot>
 </slots>
</ui>
